Vungle/VungleRouter.h

```diff
@@ -40,6 +41,7 @@ extern const CGSize kVNGLeaderboardBannerSize;
 - (void)requestRewardedVideoAdWithCustomEventInfo:(NSDictionary *)info delegate:(id<VungleRouterDelegate>)delegate;
 - (void)requestBannerAdWithCustomEventInfo:(NSDictionary *)info size:(CGSize)size delegate:(id<VungleRouterDelegate>)delegate;
 - (BOOL)isAdAvailableForPlacementId:(NSString *)placementId;
+- (NSString*)currentSuperToken;
```

Comment:
Should `- (NSString *)`, there is a space between `NSString` and `*`, thanks. :)

Reply:
Resolved.